### PR TITLE
Support Electron

### PR DESCRIPTION
**Pull request by gucong3000**

### CODE REVIEW

**Comment by gucong3000** [comment]
image
image

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - 'iojs'
-  - '0.12'
-  - '0.10'
+  - "stable"
```
Comment (SamVerschueren) [inline .travis.yml]: Revert this. We're planning to do this in bulk.

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "node test.js"
+    "test": "ava test.js"
```
Comment (SamVerschueren) [inline package.json]: `ava`

File: index.js [hunk 3]
```diff
@@ -1,5 +1,8 @@
 'use strict';
 var chalk = require('chalk');
+var isBrowser = require('is-in-browser').default;
```
Comment (SamVerschueren) [inline index.js]: I don't think we're interested in specific browser code. This should be solved by things like Webpack or others. But I'll let @sindresorhus decide on this one.

Comment (sindresorhus) [inline index.js]: Yeah, don't want it. This is for Node.js, not the browser.

Comment (gucong3000) [inline index.js]: Removed `is-in-browser`.

File: .travis.yml (hunk 1 above)
Comment (gucong3000) [inline .travis.yml]: OK

File: package.json (hunk 2 above)
Comment (gucong3000) [inline package.json]: has been rolled back

File: index.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 'use strict';
 var chalk = require('chalk');
+var os = require('os');
+var support = process.platform !== 'win32' || process.versions.electron || parseFloat(os.release()) >= 10;
```
Comment (sindresorhus) [inline index.js]: support => isSupported

parseFloat is not a good way to compare versions. They're not numbers. Instead use `String#split()` and only compare the first part.

For clarity `parseFloat(os.release()) >= 10` => `(process.platform === 'win32' && parseFloat(os.release()) >= 10)`.

File: index.js [hunk 5]
```diff
@@ -1,18 +1,19 @@
...
+var isSupported = process.platform !== 'win32' || typeof document !== 'undefined' && document.nodeType === 9;
 
 var main = {
 	info: chalk.blue('ℹ'),
-	success: chalk.green('✔'),
-	warning: chalk.yellow('⚠'),
-	error: chalk.red('✖')
+	success: chalk.green('✔️'),
+	warning: chalk.yellow('⚠️'),
+	error: chalk.red('❌️')
```
Comment (sindresorhus) [inline index.js]: Don't do unrelated changes.

File: index.js [hunk 6]
```diff
@@ -1,18 +1,19 @@
 'use strict';
 var chalk = require('chalk');
+var isSupported = process.platform !== 'win32' || typeof document !== 'undefined' && document.nodeType === 9;
```
Comment (sindresorhus) [inline index.js]: `typeof document !== 'undefined' && document.nodeType === 9` should be defined above in a separate variable first called `isBrowser`.
